### DISCUSSION TITLE
zustand error in spite of using ReactFlowProvider

### DISCUSSION THREAD

**Discussion by hyosang-park** [xyflow/xyflow#4685] [Q&A] [answered]
Please somebody help me. I tested the code below but just got the message in browser.

[React Flow]: Seems like you have not used zustand provider as an ancestor.

![image](https://github.com/user-attachments/assets/5a13bc40-9737-4765-86fb-e6b062e96949)

Thank you!

**Comment by moklick** [comment, marked as answer]
this should work. Sometimes it helps to remove node_modules and install again. If not, could you create a codesandbox or repo for this?

**Reply by hyosang-park** [reply]
Thank you for answering! I'm handling this problem in different ways.
